test/components/tree-view.spec.tsx

```diff
@@ -50,38 +44,16 @@ const changedLabel = 'Kaiserschmarrn';
 const newTreeData = JSON.parse(JSON.stringify(treeData));
 newTreeData[0].children![2].children!.push({label: changedLabel});
 
-export interface TreeViewWrapperState {
-    treeData: object[];
-}
-
-export class TreeViewWrapper extends React.Component<{}, TreeViewWrapperState> {
-    public state = {treeData};
-
-    public render() {
-        return <TreeView dataSource={this.state.treeData}/>;
-    }
+class TreeViewDemoDriver extends DriverBase {
+    public static ComponentClass = TreeViewDemo;
 
-    public switchDataSource = () => {
-        this.setState({
-            treeData: newTreeData
-        });
-    }
+    public treeView = new TreeViewDriver(() => this.select('TREE_VIEW'));
 }
 
-export class TreeViewMobxWrapper extends React.Component<{}, {}> {
-    @observable private obsTreeData: TreeItemData[] = treeData;
-
-    public render() {
-        return <TreeView dataSource={this.obsTreeData}/>;
-    }
+class TreeViewDemoCustomDriver extends DriverBase {
```

Comment:
the "custom" part is the treeView right? so maybe rename to `CustomTreeViewDemoDriver`